newsroom/templates/settings.html

```diff
@@ -16,11 +16,13 @@
         <div class='side-navigation' id='settings-menu'>
             <ul>
                 {% for app in settings_apps|sort(attribute='weight') %}
-                <li>
-                    <a class="side-navigation__btn {{ "active" if setting_type == app._id }}"
-                        href="{{ url_for('settings.app', app_id=app._id) }}">
-                        {{ gettext(app.name) }}</a>
-                </li>
+                    {% if is_admin() or ( is_admin_or_account_manager() and app.allow_account_mgr ) %}
```

Comment:
would probably simplify the check in the template and keep the logic in python, maybe something like:
```
{% if is_authorized(app._id) %}
```
but maybe we can actually do the filtering on the settings_apps before going to template

Reply:
OK, added a function to return the authorized list of apps, that simplifies things.